### PR TITLE
Add README to dendrites project

### PR DESCRIPTION
**Pull request by vkakerbeck**
Added README from https://github.com/numenta/htmpapers/blob/master/biorxiv/going_beyond_the_point_neuron/README.md to this project folder (without prerequisites section).

### CODE REVIEW

File: projects/dendrites/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Running experiments
+
+This repository contains versioned code used to replicate the experiments in the paper. For the latest pytorch code for using the code in your own projects check the [nupic.research](https://github.com/numenta/nupic.research) and [nupic.torch](https://github.com/numenta/nupic.torch) repositories.
+
+Below are instructions for reproducing all the charts and tables presented in the paper. There might be small differences due to randomness.
```
Comment (subutai) [inline projects/dendrites/README.md:11]: Can get rid of this line.

**Review by subutai — CHANGES_REQUESTED** [review]

File: projects/dendrites/README.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
 ## Running experiments
 
-This repository contains versioned code used to replicate the experiments in the paper. For the latest pytorch code for using the code in your own projects check the [nupic.research](https://github.com/numenta/nupic.research) and [nupic.torch](https://github.com/numenta/nupic.torch) repositories.
+This repository contains the code originally used in the above paper. Since it contains our latest code for dendrites, there may be differences from the paper. To reproduce the charts and results presented in the paper, please go to [htmpapers](https://github.com/numenta/htmpapers/tree/master/biorxiv/going_beyond_the_point_neuron) for a versioned copy of the code.
 
-Below are instructions for reproducing all the charts and tables presented in the paper. There might be small differences due to randomness.
+For the latest pytorch code for using the code in your own projects check the [nupic.research](https://github.com/numenta/nupic.research) and [nupic.torch](https://github.com/numenta/nupic.torch) repositories.
```
Comment (subutai) [inline projects/dendrites/README.md:11]: I think we can get rid of this sentence now.

**Review by subutai — CHANGES_REQUESTED** [review]

**Comment by lscheinkman** [comment]
@vkakerbeck @subutai We should create pull requests from your personal fork  vkakerbeck/nupic.research not branch on the upstream repository numenta/nupic.research

**Review by subutai — APPROVED** [review]

**Comment by vkakerbeck** [comment]
Oh right, sorry! I will do that next time, totally forgot about it.

**Review by karangrewal — APPROVED** [review]

**Comment by vkakerbeck** [comment]
New PR from fork here: https://github.com/numenta/nupic.research/pull/619

**Pull request closed without merging**